transforms/images/polus-imagej-filter-gauss-plugin/plugin.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "ImageJ filter gauss",
+  "version": "0.3.0",
+  "title": "ImageJ filter gauss",
+  "description": "This plugin applies a Gaussian Convolutional filter to an input collection.",
+  "author": "Benjamin Houghton (benjamin.houghton@axleinfo.com)",
+  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+  "repository": "https://github.com/labshare/polus-plugins",
+  "website": "https://ncats.nih.gov/preclinical/core/informatics",
+  "citation": "",
+  "containerId": "polusai/imagej-filter-gauss-plugin:0.3.0",
+  "inputs": [
+    {
+      "name": "opName",
+      "type": "enum",
+      "options": {
+        "values": ["GaussRAISingleSigma",
+                    "DefaultGaussRAI"]
+      },
+      "description": "Operation overloading method to perform",
+      "required": false
+    },
+    {
+      "name": "inpDir",
+      "type": "collection",
+      "description": "Collection to be processed by this plugin",
+      "required": false
+    },
```

Comment:
I have tested this in WIPP and it seems they can be set to `True`. The `opName` should be required for all ops. However, the first input might not always be required for all ops. As of now since only ops that process images are generated than most likely it will be required for each overloading operation. This is a modification that could be made to the pipeline. For this plugin I have set these to `True`